src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -54,12 +53,12 @@
     private static final String TAG_MAX_CITIZENS               = "maxCitizens";
     private static final String TAG_BUILDINGS                  = "buildings";
     private static final String TAG_CITIZENS                   = "citizens";
-    private static final String TAG_ACHIEVEMENT            = "achievement";
-    private static final String TAG_ACHIEVEMENT_LIST       = "achievementlist";
-    private static final String TAG_WORK                   = "work";
-    private static final String TAG_MANUAL_HIRING          = "manualHiring";
-    private static final String TAG_WAYPOINT               = "waypoints";
-    private static final String TAG_BLOCK                  = "blockState";
+    private static final String TAG_ACHIEVEMENT                = "achievement";
+    private static final String TAG_ACHIEVEMENT_LIST           = "achievementlist";
+    private static final String TAG_WORK                       = "work";
+    private static final String TAG_MANUAL_HIRING              = "manualHiring";
+    private static final String TAG_WAYPOINT                   = "waypoints";
+    private static final String TAG_BLOCK                      = "blockState";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TAG_BLOCK" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)